src/7fd10_len_b40.c

```diff
@@ -16,7 +16,11 @@ BSS s16 D_8010CCFA;
 BSS s16 D_8010CCFC;
 BSS s16 D_8010CCFE;
 BSS s16 D_8010CD00;
+#if VERSION_PAL
+BSS char D_8010CD04[0x8];
```

Comment:
Is this all just 16-byte alignment for `D_8010CD10`?

Reply:
Probably, but it's difficult to split up the BSS stuff so that'd be a good separate effort if we wanna try that

Reply:
nm ididid